docs/static/settings-file.asciidoc

```diff
@@ -116,16 +116,12 @@ increasing this number to better utilize machine processing power.
 
 | `pipeline.ordered`
 a|
-Set the pipeline event ordering.Valid options are:
+Set the pipeline event ordering. Valid options are:
 
-* `auto`
-* `true`
-* `false`
-
-`auto` will  automatically enable ordering if the `pipeline.workers` setting is also set to `1`.
-`true` will enforce ordering on the pipeline and prevent logstash from starting
+* `auto`. Automatically enables ordering if the `pipeline.workers` setting is also set to `1`.
```

Comment:
Should we say what happens if `pipeline.workers` is set to > 1? 

Something like 

```suggestion
* `auto`. Automatically enables ordering if the `pipeline.workers` setting is set to `1`, and disables otherwise
```